generic-sensor/resources/generic-sensor-helpers.js

```diff
@@ -92,6 +93,11 @@ function verifyGeoSensorReading(pattern, {latitude, longitude, altitude,
     accuracy, altitudeAccuracy, heading, speed], timestamp, isNull);
 }
 
+function verifyProSensorReading(pattern, {distance, max, near, timestamp},
+  isNull) {
```

Comment:
There's no need for this line break here.

Reply:
Done.